docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,199 +1,244 @@
 ---
-title: Getting Started with Desktop Access
-description: Passwordless access to Windows systems for local users
+title: Get started with Desktop Access
+description: Configure passwordless access to Windows computers for local users.
 videoBanner: 9DyKQbg4ORc
 ---
 
-This guide will help you configure Teleport to provide secure, passwordless access
-to Windows desktops. This configuration does not require an Active Directory domain.
+This guide demonstrates how to configure Teleport to provide secure, passwordless access
+to Microsoft Windows desktops that aren't joined to an Active Directory domain.
+Passwordless access is only supported for **commercial editions** of Teleport.
```

Comment:
```suggestion

Passwordless access is only supported for **commercial editions** of Teleport.
```
I don't think this belongs in the first paragraph, since it's more of a caveat than a statement about the whole guide.